package/kernel/linux/modules/netsupport.mk

```diff
@@ -1476,6 +1476,21 @@ endef
 
 $(eval $(call KernelPackage,xdp-sockets-diag))
 
+define KernelPackage/inet-mptcp-diag
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=INET diag support for MultiPath TCP
+  DEPENDS:= @KERNEL_MPTCP @KERNEL_MPTCP_IPV6 +kmod-inet-diag
```

Comment:
I don't see this depending on `MPTCP_IPV6` in the kernel, just `INET_DIAG` and implicitly `MPTCP`.